en/docs/reference/connectors/google-firebase-connector/google-firebase-connector-example.md

```diff
@@ -105,12 +103,11 @@ Follow these steps to set up the Integration Project and import Google Firebase
                 </log>
                 <respond/>
             </inSequence>
-            <outSequence/>
             <faultSequence/>
         </resource>
     </api>
    ```
-3. Right click on the created Integration Project and select, -> **New** -> **Sequence** to create a sequence. Here we will define the logic how the push notification should be constructed. You can extract the information from the incoming HTTP message and set to the properties so that they will be picked up by the connector to construct push notification message. All the fields are not mandatory - some are specific to Android devices and some are specific to Web apps.  Note how this sequence is called by the API. 
+3.Create a new sequence, defining the logic how the push notification should be constructed. You can extract the information from the incoming HTTP message and set to the properties so that they will be picked up by the connector to construct push notification message. All the fields are not mandatory - some are specific to Android devices and some are specific to Web apps.  Note how this sequence is called by the API. 
```

Comment:
```suggestion
3.Create a new sequence, defining the logic how the push notification should be constructed. You can extract the information from the incoming HTTP message and set to the properties so that they will be picked up by the connector to construct push notification message. All the fields are not mandatory - some are specific to Android devices and some are specific to Web apps. Note how this sequence is called by the API. 
```